notebooks/official/sdk/sdk2_bigframes_sklearn.ipynb

```diff
@@ -0,0 +1,725 @@
+{
```

Comment:
<p>Update the folder in the link from "training"  to "sdk".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>